packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -54,7 +55,7 @@ function FeedNav(): ReactElement {
     >
       {isMobile && <MobileFeedActions />}
       <TabContainer
-        controlledActive={notifications ? 'notifications' : undefined}
+        shouldMatchUrl
```

Comment:
I think we can listen to `useRouter.pathname` and pass that as `controlledActive`? We can make a map `tabUrlToLabel` and then use it below for `controlledActive` and also `Tab`.

This would add no new code to `TabContainer` and also avoid adding new hook.

Reply:
+1 was trying point to how we do these kind of checks currently.

Reply:
I did it like that before, but there was a visible lag when the route change was taking more time

Reply:
I would like to see it on preview, where pages are prerender? Because I think it will not be as near as bad as in dev.

Reply:
Either way I don't think we need to add internal state to `TabContainer`, it will just cause problems down the line when you have too props that cancel each other.

Reply:
There already was internal state that controlled active tab. Let me create another PR with controlledActive and we can see how it behaves in preview :)

Reply:
Here's the other approach: 
- https://github.com/dailydotdev/apps/pull/2963